svg/svg.go

```diff
@@ -92,7 +94,45 @@ func NewWindowControls(r float64, x, y float64) *etree.Element {
 		circle.CreateAttr("fill", color)
 		bar.AddChild(circle)
 	}
-	return bar
+	controlsWidth := float64(len(bar.ChildElements()))*float64(x) + float64(r)*2
```

Comment:
controlWith depends on bar only, so this code could be moved out the function to the place you called the method